pkgs/by-name/co/config-store/package.nix

```
@@ -0,0 +1,30 @@
+{
+  rustPlatform,
+  fetchFromGitHub,
+  lib,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "config-store";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "DOD-101";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    sha256 = "sha256-dmFIB9tVI5/hnI+VKeawFzKi6UJrRis0tpeQE5a5dGU=";
```

```suggestion
    tag = "v${version}";
    hash = "sha256-dmFIB9tVI5/hnI+VKeawFzKi6UJrRis0tpeQE5a5dGU=";
```
This use of `rev` can now be replaced with the `tag` argument. We should also prefer the generic `hash` argument rather than any specific algo

---

Interesting. I didn't know that. Thx